examples/s/d/nn/_project/aggregate_surfaces.py

```diff
@@ -82,8 +78,8 @@ def main():
 
         # ==============================================================================
         # Example 2: We only want the metadata (e.g. we are in a cloud service)
-        metadata = exp.generate_metadata(aggregated_surface, operation=operation)
-        print(f"Example 2: Metadata generated")
+        exp.generate_metadata(aggregated_surface, operation=operation)
```

Comment:
Even if the `metadata` variable is not further used, I think it is important for the example to show that `generate_metadata()` returns metadata.

Reply:
Thanks, unsure why i removed this. Clearly wrong.